src/components/screen/Setting/ChangePwView.tsx

```diff
@@ -0,0 +1,96 @@
+import { Alert, KeyboardAvoidingView, SafeAreaView } from 'react-native';
+import {
+  DefaultTheme, withTheme,
+} from 'styled-components/native';
+import { InnerContainer, ModalCloseButton, ModalHeader, ModalTitle, StyledTextInput } from './styles';
+import React, { useState } from 'react';
+
+import Button from '../../shared/Button';
+import { Ionicons } from '@expo/vector-icons';
+import { getString } from '../../../../STRINGS';
+
+export interface Props {
+    theme: DefaultTheme;
```

Comment:
This is currently deprecated. We are using `const theme = useThemeContext()`.

Reply:
I'd remove it.